Tests/Marketplace/marketplace_services.py

```diff
@@ -1677,6 +1695,20 @@ def format_metadata(self, user_metadata, pack_content_items, integration_images,
         finally:
             return task_status
 
+    def pack_created_in_time_delta(self, time_delta: timedelta, index_folder_path: str) -> bool:
```

Comment:
```suggestion
    def pack_created_in_time_delta(self, time_delta: timedelta, index_folder_path: str) -> bool:
```
As this function is an indicator, I think a better naming for it would be `is_created_in_time_interval` or something similar.